src/System.Runtime.Extensions/src/System/Collections/Comparer.cs

```diff
@@ -17,7 +17,6 @@
 
 namespace System.Collections
 {
-    [Serializable]
```

Comment:
Are all of the non-generic collections still serializable after this is removed?